src/libraries/System.Private.CoreLib/src/System/Text/StringBuilder.cs

```diff
@@ -1179,6 +1179,15 @@ public StringBuilder Append(char value)
                 return this;
             }
 
+            unsafe
```

Comment:
Why is unsafe necessary?

Reply:
With Marek's suggestion below it's now necessary 😄

Reply:
We shouldn't take the suggestion.